integration_test/third_party_apps_test/applications/cassandra/centos_rhel/install

```diff
@@ -27,5 +27,9 @@ sudo mv cassandra.repo /etc/yum.repos.d/cassandra.repo
 sudo yum -y update
 
 sudo yum -y install java-1.8.0-openjdk java-1.8.0-openjdk-devel cassandra
+
+if [[ "$ID" == "rocky" && "$MAJOR_VERSION_ID" == "8" && "$(uname -m)" == "aarch64" ]]; then
+    sudo sed -i 's/-Xss256k/-Xss512k/' /etc/cassandra/default.conf/jvm-server.options
```

Comment:
That's wild -- can you add a comment with some context, i.e. this is about the OOTB stack size being too small on ARM correct?